CMakeLists.txt

```
@@ -15,31 +15,22 @@
 ###############################################################################
 # CMake build rules for Fast DDS Statistics Backend
 ###############################################################################
-cmake_minimum_required(VERSION 3.5)
-cmake_policy(VERSION 3.5...3.13)
-
-###############################################################################
-# Version
-###############################################################################
-set(PRODUCT_MAJOR_VERSION 0)
-set(PRODUCT_MINOR_VERSION 1)
-set(PRODUCT_PATCH_VERSION 0)
-set(PRODUCT_VERSION_STR ${PRODUCT_MAJOR_VERSION}.${PRODUCT_MINOR_VERSION}.${PRODUCT_PATCH_VERSION})
+cmake_minimum_required(VERSION 3.10)
+cmake_policy(VERSION 3.10...3.20)
 
 ###############################################################################
 # Project
 ###############################################################################
-project(fastdds_statistics_backend)
-set(PROJECT_VERSION_MAJOR "${PRODUCT_MAJOR_VERSION}")
-set(PROJECT_VERSION_MINOR "${PRODUCT_MINOR_VERSION}")
-set(PROJECT_VERSION_PATCH "${PRODUCT_PATCH_VERSION}")
-set(PROJECT_VERSION "${PRODUCT_VERSION_STR}")
+set(PROJECT_NAME "fastdds-statistics-backend")
+string(TOUPPER "${PROJECT_NAME}" PROJECT_NAME_UPPER)
+string(REPLACE "-" "_" PROJECT_NAME_UNDERSCORED "${PROJECT_NAME}")
```

Why not leaving project name with underscores directly?

```suggestion
set(PROJECT_NAME "fastdds_statistics_backend")
string(TOUPPER "${PROJECT_NAME}" PROJECT_NAME_UPPER)
```

---

I discussed it with @richiprosima and we agreed it's ugly 👺